flixel/input/gamepad/ButtonID.hx

```diff
@@ -0,0 +1,57 @@
+package flixel.input.gamepad;
+
+import flixel.system.macros.FlxMacroUtil;
+
+/**
+ * Maps enum values and strings to integer keycodes.
+ */
+@:enum
+abstract ButtonID(Int) from Int to Int
+{
+	public static var fromStringMap(default, null):Map<String, ButtonID>
+		= FlxMacroUtil.buildMap("flixel.input.gamepad.ButtonID");
+		
+	public static var toStringMap(default, null):Map<ButtonID, String>
+		= FlxMacroUtil.buildMap("flixel.input.gamepad.ButtonID", true);
+	
+	// Button Indices
+	var ANY               = -2;		
+	var NONE              = -1;		
+	var A                 =  0;		//BOTTOM face button
+	var B                 =  1;		//RIGHT  face button
+	var X                 =  2;		//LEFT   face button
+	var Y                 =  3;		//TOP    face button
+	var LEFT_SHOULDER     =  4;		//left "bumper"
+	var RIGHT_SHOULDER    =  5;		//right "bumper"
+	var BACK              =  6;		//a.k.a "select", left center button
+	var START             =  7;		//right center button
+	var LEFT_STICK_BTN    =  8;		//Click left stick
```

Comment:
Okay, but what's the analog `LEFT_TRIGGER` doing here, it's an axis and not a button, right?